test/fail_compilation/fail13902.d

```diff
@@ -57,7 +57,7 @@ fail_compilation/fail13902.d(88): Error: Using the result of a comma expression
 fail_compilation/fail13902.d(75): Error: returning `& x` escapes a reference to parameter `x`
 fail_compilation/fail13902.d(76): Error: returning `&s1.v` escapes a reference to parameter `s1`
 fail_compilation/fail13902.d(81): Error: returning `& sa1` escapes a reference to parameter `sa1`
-fail_compilation/fail13902.d(82): Error: returning `&sa2[0][0]` escapes a reference to parameter `sa2`
+fail_compilation/fail13902.d(82): Error: returning `& sa2` escapes a reference to parameter `sa2`
```

Comment:
off topic: why does `&` is formatted sometimes with a trailing space and sometimes not ?

Reply:
The parser generates an `AddrExp` for `&x`, which gets printed as `&x` as well. Semantic analysis turn an `AddrExp` it into a `SymOffExp` with offset 0, which gets printed as `& x`. (See dmd.hdrgen.d for the relevant source code)

Reply:
shouldn't this be considered as an issue ? What's the point ?

Reply:
I mean, why the space ?

Reply:
I don't know why the space was added, but from a compiler developer perspective, it is useful to distinguish an `AddrExp` or `SymOff` expression.

Reply:
that space should be removed  IMO.